save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/IndexLogoButtons.kt

```diff
@@ -10,55 +10,67 @@ import react.ChildrenBuilder
 import react.VFC
 import react.dom.html.ReactHTML.button
 import react.dom.html.ReactHTML.div
-import react.dom.html.ReactHTML.h3
+import react.dom.html.ReactHTML.h4
 import react.dom.html.ReactHTML.img
 import react.dom.html.ReactHTML.span
 import react.router.dom.Link
-import web.cssom.Border
-import web.cssom.ClassName
-import web.cssom.rem
+import web.cssom.*
 
 val logoButtons = VFC {
```

Comment:
Fun fact: in new `kotlin-wrappers` there is no `VFC`.

Reply:
hahahahahahaa

Reply:
As I said - no need to be so proactive

Reply:
Still, their solution is more convenient: if you do not pass `Props`, it is considered to be `Props` by default. (Migration: `VFC` -> `FC`)